src/ng/directive/ngModel.js

```diff
@@ -570,34 +592,34 @@ var NgModelController = ['$scope', '$exceptionHandler', '$attrs', '$element', '$
     processAsyncValidators();
 
     function processParseErrors() {
```

Comment:
That would require a slightly bigger change because these methods depend on local state defined within `$$runValidators` (such as `doneCallback`). Definitely could be done though...